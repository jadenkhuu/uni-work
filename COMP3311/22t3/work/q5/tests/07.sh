python3 ./q5 Caulfield 2020-07-25
