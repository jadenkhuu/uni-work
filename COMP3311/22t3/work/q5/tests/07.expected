Race meeting at Caulfield on 2020-07-25

Race 1, prize pool $1000, run over 2400m
Liam Neighson ridden by Tommy Lowrey wins $700
Salmon Trout ridden by Jose Lezcano wins $200
Chatters ridden by Johnny Murtagh wins $100

Race 2, prize pool $6000, run over 2100m
Schumer ridden by Bob Skelton wins $4200
Mane Attraction ridden by Peter Schiergen wins $1200
Papyrus ridden by Brian Hernandez Jr. wins $600

Race 3, prize pool $8000, run over 1800m
Justine Thyme ridden by Jean Cruguet wins $5600
Pinto Bean ridden by Paddy Brennan wins $1600
Knight of the Thistle ridden by Rajiv Maragh wins $800

Bletchingly Stakes, prize pool $160000, run over 1200m
Brown Jack ridden by Campbell Gillies wins $112000
Coronach ridden by Chris Hayes wins $32000
Lemberg ridden by Johnny Loftus wins $16000
